### PR TITLE
[Ticket/6] Travis checks

### LINKED ISSUE #6: Travis Checks
We should add at least basic Travis Extension tests, such as EPV check to this extension as well.

### PR DESCRIPTION
**Pull request by gn36**
This PR adds travis checks to the extension - Extension Pre Validator as well as one very basic functional test to see if an update completely breaks display on viewtopic.php (no test for correct handling of links, this will require additional tests). 

Please activate the travis check for this extension (https://travis-ci.org/profile/phpbb-de)  before merging so we can actually see whether I broke something :-)

Fixes #6

### CODE REVIEW

File: tests/functional/install_test.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+*
+* @package phpBB.de Pastebin
```
Comment (Crizz0) [inline tests/functional/install_test.php]: Änderst du das noch auf die aktuelle Extension?

File: tests/functional/install_test.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* @copyright (c) 2014 phpBB.de
```
Comment (Crizz0) [inline tests/functional/install_test.php]: 2015?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+language: php
+sudo: required
```
Comment (nickvergessen) [inline .travis.yml:2]: sudo must be first

Comment (gn36) [inline .travis.yml:2]: Why is that? I don't really see a reason. It works fine like this, the documentation states it must be first level, not the first line in the file.

**Pull request merged**